src/gui/ColorManager.cpp

```diff
@@ -96,6 +97,8 @@ void ColorManager::setColorBlind(ColorBlindMode mode) {
             ColorManager::plasmaSightsOffColor = 0xff008e00;
             ColorManager::plasmaSightsOnColor = 0xffff2600;
             ColorManager::shieldGaugeColor = 0xff0053b4;
+            ColorManager::hudColor = 0xff03f5f5;
+            ColorManager::hudCriticalColor = 0x22fa1313;
```

Comment:
You should remove these lines! Since both these colors are now modifiable by prefs, color blind mode should never have an effect on them. With these lines in place, `/p colorBlindMode 0` will apply these defined colors rather than fetch them from prefs.

Reply:
They are removed